cypress/support/kaoto-ui-commands/default.js

```diff
@@ -1,95 +1,183 @@
 });
 
 Cypress.Commands.add('openSettingsModal', () => {
-    cy.openMenuDropDown();
-    cy.get('[data-testid="kaotoToolbar-kebab__settings"]').click();
+  cy.openMenuDropDown();
+  cy.get('[data-testid="kaotoToolbar-kebab__settings"]').click();
 });
 
 Cypress.Commands.add('closeAppearanceModal', () => {
-    cy.get('[data-ouia-component-id="appearance-modal"] > button[aria-label="Close"]').click();
+  cy.get('[data-ouia-component-id="appearance-modal"] > button[aria-label="Close"]').click();
 });
 
 Cypress.Commands.add('closeAboutModal', () => {
-    cy.get('.pf-c-button.pf-m-plain').click();
+  cy.get('.pf-c-button.pf-m-plain').click();
 });
 
 Cypress.Commands.add('closeMenuModal', () => {
-    cy.get('[data-ouia-component-id="settings-modal"] > button[aria-label="Close"]').click();
+  cy.get('[data-ouia-component-id="settings-modal"] > button[aria-label="Close"]').click();
 });
 
 Cypress.Commands.add('cancelMenuModal', () => {
-    cy.get('[data-testid="settings-modal--cancel"]').click();
+  cy.get('[data-testid="settings-modal--cancel"]').click();
 });
 
 Cypress.Commands.add('saveMenuModal', (integrationChanged) => {
-    integrationChanged = integrationChanged ?? false;
-    cy.get('[data-testid="settings-modal--save"]').click();
-    cy.get('.pf-c-alert').should('contain.text', 'Configuration settings saved successfully.');
-    cy.get('.pf-c-alert__action').children('button').click();
-    cy.wait('@getIntegration');
-    if (integrationChanged) {
-        cy.waitVisualizationUpdate();
-        cy.wait('@getDSLs');
-    }
+  integrationChanged = integrationChanged ?? false;
+  cy.get('[data-testid="settings-modal--save"]').click();
+  cy.get('.pf-c-alert').should('contain.text', 'Configuration settings saved successfully.');
+  cy.get('.pf-c-alert__action').children('button').click();
+  cy.wait('@getIntegration');
+  if (integrationChanged) {
+    cy.waitVisualizationUpdate();
+    cy.wait('@getDSLs');
+  }
 });
 
 Cypress.Commands.add('switchAppearanceTheme', (object) => {
-    object = object ?? '';
-    if (object === 'editor') {
-        cy.get('[data-testid="appearance--theme-editor-switch"]').click({ force: true });
-    } else {
-        cy.get('[data-testid="appearance--theme-ui-switch"]').click({ force: true });
+  object = object ?? '';
+  if (object === 'editor') {
+    cy.get('[data-testid="appearance--theme-editor-switch"]').click({ force: true });
+  } else {
+    cy.get('[data-testid="appearance--theme-ui-switch"]').click({ force: true });
+  }
+});
+
+/**
+ * Select from integration type dropdown
+ * Possible values are - Integration, Camel Route, Kamelet, KameletBinding
+ */
+Cypress.Commands.add('switchIntegrationType', (type) => {
+  cy.get('[data-testid="dsl-list-dropdown"]').click({ force: true });
+
+  cy.get('.pf-c-menu__item-text')
+    .contains(type)
+    .then((element) => {
+      cy.wrap(element).click();
+    });
+  cy.get('.pf-c-modal-box__footer').children('.pf-c-button.pf-m-primary').click();
```

Comment:
Would it be convenient to have a better handler for the `Accept` button? I can add it :smiley:

Reply:
should be ok now